src/sidebar/components/ToastMessages.tsx

```diff
@@ -16,10 +18,23 @@ function ToastMessages({ toastMessenger }: ToastMessageProps) {
   const messages = store.getToastMessages();
 
   return (
-    <BaseToastMessages
-      messages={messages}
-      onMessageDismiss={(id: string) => toastMessenger.dismiss(id)}
-    />
+    <div
+      className={classnames(
+        // Set an aggressive z-index as we want to ensure toast messages are
+        // rendered above other content
```

Comment:
```suggestion
        // Ensure toast messages are rendered above other content
```

This shorter version seems clear enough.